src/crimson/os/seastore/seastore.cc

```diff
@@ -69,7 +69,7 @@ void SeaStore::register_metrics()
       "seastore",
       {
         sm::make_histogram(
-          "op_lat", [this, op_type] {
+          "op_lat", [this, op_type=op_type] {
```

Comment:
@cyx1231st i've fix this issue couple times. probably it is noteworthy that we cannot capture a label using capture list without defining a variable?

Reply:
Let me remind @liu-chunmei 

But I have to admit it is very easy to miss this issue as gcc can compile without any warning.